lib/prompt.ts

```diff
@@ -206,30 +206,38 @@ const extractSystemPrompt = `You are extracting content on behalf of a user. You
 
 Print the exact text from the DOM elements with all symbols, characters, and endlines as is.
 Print null or an empty string if no new information is found.
-
-ONLY print the content using the print_extracted_data tool provided.
-ONLY print the content using the print_extracted_data tool provided.
 `;
 
-export function buildExtractSystemPrompt(): ChatMessage {
-  const content = extractSystemPrompt.replace(/\s+/g, " ");
+export function buildExtractSystemPrompt(llmClient: LLMClient): ChatMessage {
+  let content = extractSystemPrompt.replace(/\s+/g, " ");
+  if (llmClient.type === "anthropic") {
```

Comment:
not a fan of this approach -- would prefer that `buildExtractSystemPrompt` becomes a method of `llmClient`, and `AnthropicClient` implements this there.